spec/features/users/user_creates_new_account_spec.rb

```diff
@@ -6,14 +6,12 @@
 
     fill_in "user_name", with: "Mary"
     fill_in "user_email", with: "Mary@example.com"
+    fill_in "user_email_confirmation", with: "Mary@example.com"
     fill_in "user_password", with: "secret"
     click_on I18n.t("helpers.submit.user.create")
 
-    expect(page).to have_text(I18n.t("sessions.new.title"))
+    expect(page).to have_text(I18n.t("users.show.welcome"))
     expect(page).
-      to have_text(I18n.t(
-        "flash.actions.create.notice",
-        resource_name: "User"
-      ))
+      to have_text(I18n.t("sessions.new.flash.alert"))
```

Comment:
Do we also want to check that the `current_path` is the user profile page?
i.e. `(expect(current_path).to eq user_path`

Reply:
Done in sha: 1159cb8